add-on/chrome/bootstrap.js

```diff
@@ -604,24 +604,25 @@ var WindowListener = {
         }
 
         let browser = gBrowser.selectedBrowser;
-
-        let cursorContainer = document.getElementById("loop-remote-cursor-container");
-        if (!cursorContainer) {
-          cursorContainer = document.createElement("img");
+        let cursor = document.getElementById("loop-remote-cursor");
+        if (!cursor) {
+          // Create a 100% size container to keep the pointer inside.
+          // This allows us to hide the overflow when out of bounds.
+          let cursorContainer = document.createElement("div");
           cursorContainer.setAttribute("id", "loop-remote-cursor-container");
-          let cursor = document.createElement("img");
+
+          cursor = document.createElement("img");
           cursor.setAttribute("id", "loop-remote-cursor");
           cursorContainer.appendChild(cursor);
 
           browser.parentNode.appendChild(cursorContainer);
         }
 
-        // Update the cursor's position. Cursor container (browser.parent)
-        // is a xul:stack, so positioning with left/top works.
-        cursorContainer.setAttribute("left",
-                            cursorData.ratioX * browser.boxObject.width);
-        cursorContainer.setAttribute("top",
-                            cursorData.ratioY * browser.boxObject.height);
+        // Update the cursor's position with CSS.
```

Comment:
I'd suggest leaving the original comment; it gives the reader more mental context about what's going on here.